### DISCUSSION TITLE
Issue with kernel installation on Debian 11 - cdrom

### DISCUSSION THREAD

**Discussion by dewey-dewey** [linux-surface/linux-surface#660] [Q&A]
Hi all- 

Working on a Dual Booted Surface Book 2 running Win10 and Deb11. Used the linux-surface kernel on a previous Ubuntu install, but am hitting a snag currently.

Following the steps in the install guide:

`$ wget -qO - https://raw.githubusercontent.com/linux-surface/linux-surface/master/pkg/keys/surface.asc     | gpg --dearmor | sudo dd of=/etc/apt/trusted.gpg.d/linux-surface.gpg
`
returns:

`4+1 records in
4+1 records out
2233 bytes (2.2 kB, 2.2 KiB) copied, 0.559704 s, 4.0 kB/s`

and:

`$ echo "deb [arch=amd64] https://pkg.surfacelinux.com/debian release main" \
        | sudo tee /etc/apt/sources.list.d/linux-surface.list`

`deb [arch=amd64] https://pkg.surfacelinux.com/debian release main`

which seems fine. But the next step, I start getting some errors.

`$ sudo apt update`

`Ign:1 cdrom://[Debian GNU/Linux 11.1.0 _Bullseye_ - Official amd64 DVD Binary-1 20211009-10:08] bullseye InRelease
Err:2 cdrom://[Debian GNU/Linux 11.1.0 _Bullseye_ - Official amd64 DVD Binary-1 20211009-10:08] bullseye Release
  Please use apt-cdrom to make this CD-ROM recognized by APT. apt-get update cannot be used to add new CD-ROMs
Hit:3 http://deb.debian.org/debian bullseye InRelease
Hit:4 http://security.debian.org/debian-security bullseye-security InRelease   
Hit:5 http://deb.debian.org/debian bullseye-updates InRelease                  
Get:6 https://pkg.surfacelinux.com/debian release InRelease [1,830 B]       
Get:7 https://pkg.surfacelinux.com/debian release/main amd64 Packages [5,770 B]
Reading package lists... Done         
E: The repository 'cdrom://[Debian GNU/Linux 11.1.0 _Bullseye_ - Official amd64 DVD Binary-1 20211009-10:08] bullseye Release' does not have a Release file.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.`

Any guidance to next steps? This is something I haven't encountered yet.

Thanks.

**Comment by dorianclay** [comment]
Fix by updating `ca-certificates`:
```bash
sudo apt update
sudo apt upgrade ca-certificates
# update again, now to get surfacelinux
sudo apt update
```

See [this comment][1]. 

[1]: https://github.com/linux-surface/linux-surface/issues/625#issuecomment-973365972
